### PR TITLE
Ginga viewer with matplotlib backend addition, this updates the module to v0.5

### PR DESCRIPTION
**Pull request by sosey**

### CODE REVIEW

File: imexam/connect.py [hunk 1]
```diff
@@ -50,20 +59,45 @@ class Connect(object):
 
     """
 
-    def __init__(self, target=None, path=None, viewer="ds9", wait_time=10, quit_window=True):
+    def __init__(self, target=None, path=None, viewer="ds9",
+                 wait_time=10, quit_window=True):
+
+        # better dynamic way so people can add their own viewers?
+        _possible_viewers = ["ds9"]
```
Comment (embray) [inline imexam/connect.py]: Yeah, this is obv. fine for now, but it would be interesting to extend this in the future.

File: imexam/ds9_viewer.py [hunk 2]
```diff
@@ -333,44 +342,30 @@ def _set_frameinfo(self):
...
 
-            return valid
+            if self._viewer[frame]['filename']:
+                return True
+
+            else:
+                try:
+                    if self._viewer[frame]['user_array'].any():
+                        return True
+                except AttributeError as ValueError:
```
Comment (embray) [inline imexam/ds9_viewer.py]: Not sure what's going on here...  Was this supposed to be `except (AttributeError, ValueError):` ?

File: imexam/connect.py (hunk 1 above)
Comment (sosey) [inline imexam/connect.py]: I'm unsure what you mean?

File: imexam/ds9_viewer.py (hunk 2 above)
Comment (sosey) [inline imexam/ds9_viewer.py]: huh, I'm not sure what happened there, maybe something got mistranslated when I was merging code from few branches, I'll check it out.

**Pull request merged**